### PR TITLE
nanomed update

### PR DESCRIPTION
**Pull request by pudl**
in order to help doctors keep up, since they now need a lot more to heal people, the nanomeds have been stocked with a lot of extra stuff.
12x syringe
10x styptic patch
10x silver sulf patch
4x charcoal bottle
4x epinephrine bottle
4x morphine bottle
4x diphen bottle
3x potassium iodide bottle
5x salglu bottle
3x atropine bottle
6x spaceacillin syringe
10x calomel syringe
10x salbu pill
10x mannitol pill
5x mutadone pill
4x health analyzer
2x crew monitor
CONTRABAND:
all of the same except with 3 bottles of toxin

the two types of wallmed have been combined into one wallmed, which contains all of the above except halved. in instances where the number was odd, the higher of the two numbers was taken.

medical closet has been stocked with a lot of new stuff.
2x beakers
2x droppers
1x medical belt
1x syringe box
1x toxin bottle
2x morphine bottles
3x epi bottles
3x charcoal bottles
1x rxglasses box

this, of course, 3 new pills and 3 new bottles in code so they could be in the nanomed.

gives genetics a bottle of mutadone pills on box and eff3 and puts a public bottle of epinephrine on the medbay desk. adds a single syringe to boxstation's medical storage.

medkits now have some new equipment.

normal
2x styptic patch
2x sulf patch
1x salicyclic pill
1x health analyzer

fire
4x silver sulf patches
1x health analyzer
1x epipen
1x salicyclic pill

toxin is unchanged

oxygen
4x salbu pills
2x epipen
1x health analyzer

all maps and z levels are updated for this.

renames styptic patch to brute patch and silver sulf patch to burn patch. renames charcoal bottle to antitoxin bottle

adds myself to admins.txt

### CODE REVIEW

File: code/modules/reagents/reagent_containers/patch.dm [hunk 1]
```diff
@@ -22,7 +22,7 @@
 	return 1 // Masks were stopping people from "eating" patches. Thanks, inheritance.
 
 /obj/item/weapon/reagent_containers/pill/patch/styptic
-	name = "styptic powder patch"
```
Comment (Iamgoofball) [inline code/modules/reagents/reagent_containers/patch.dm]: i say keep it as styptic

File: code/modules/reagents/reagent_containers/patch.dm [hunk 2]
```diff
@@ -30,7 +30,7 @@
 	reagents.add_reagent("styptic_powder", 50)
 
 /obj/item/weapon/reagent_containers/pill/patch/silver_sulf
-	name = "silver sulfadiazine patch"
```
Comment (Iamgoofball) [inline code/modules/reagents/reagent_containers/patch.dm]: i say keep it as silver sulf

File: code/modules/reagents/reagent_containers/pill.dm [hunk 3]
```diff
@@ -146,8 +146,26 @@
...
+	icon_state = "pill5"
 
 /obj/item/weapon/reagent_containers/pill/epinephrine/New()
 	..()
-	reagents.add_reagent("epinephrine", 50)
\ No newline at end of file
+	reagents.add_reagent("epinephrine", 50)
+
+/obj/item/weapon/reagent_containers/pill/mannitol
+	name = "epinephrine pill"
```
Comment (Iamgoofball) [inline code/modules/reagents/reagent_containers/pill.dm]: wrong name

File: code/modules/reagents/reagent_containers/patch.dm (hunk 1 above)
Comment (pudl) [inline code/modules/reagents/reagent_containers/patch.dm]: no

File: code/modules/reagents/reagent_containers/patch.dm (hunk 2 above)
Comment (pudl) [inline code/modules/reagents/reagent_containers/patch.dm]: no

**Comment by Tenebrosity** [comment]
Good idea, but a workaround at best

**Comment by AlexCrimson** [comment]
This is way too much stuff. Chemists are supposed to make chemicals, you know. Just put bruise packs and ointment back in Medkits.

**Comment by pudl** [comment]
this is why you're on the line for getting banned off this github, alex

**Comment by Iamgoofball** [comment]
@AlexCrimson we tried letting them make it

the chemists just started saying "go to the bar and drink milk for brute, OJ for oxy, tomato juice for burn, lime juice for toxin"

**Comment by pudl** [comment]
and combined with that there's the fact that there's fucking 30 other chemicals that DONT come in medbay by default? jesus fucking christ

File: code/modules/reagents/reagent_containers/patch.dm (hunk 1 above)
Comment (paprka) [inline code/modules/reagents/reagent_containers/patch.dm]: brute please

Comment (pudl) [inline code/modules/reagents/reagent_containers/patch.dm]: changed :+1:

**Comment by pudl** [comment]
shit i forgot to mapmerge, 1 sec

**Comment by pudl** [comment]
why is it merge conflict

**Comment by AlexCrimson** [comment]
Why would i get banned from here? I havent said anything unreasonable, nor have i been warned for anything.

Nanomeds are not supposed to have virtually every chemical stocked. Just the stuff to cure the basic 3 forms of damage Brute/Burn/Toxin. Maybe Chemists dont make all the other shit because goonmed was only recently added and they dont like the sudden change? Give them time. Not like it matters considering the poll is heavily in favor of reverting the goonmed stuff anyways.

**Comment by pudl** [comment]
lmao

**Comment by Firecage** [comment]
I feel that the Nanomed has WAY to much stuff there.

**Comment by Fox-McCloud** [comment]
While I do agree that this is likely overkill on supplies, it's probably also necessary, especially in the beginning. Once things die down and people get used to the new system, if it's still too much, some of the things can be removed.

The reason it's so fully stocked on Goon is that, generally speaking, chemistry is an experimental science job that serves its own crazy interests; the previous chemistry system (StarTrek chem) is the polar opposite; it's a static system designed to fulfill medbay's interests.

I really don't see or have a problem with this; but it's definitely needed until the transition is fully complete.

**Comment by Iamgoofball** [comment]
as the primary goonchem nerd

:+1: merge posthaste

**Comment by MrPerson** [comment]
>\>this is why you're on the line for getting banned off this github, alex

This attitude is atrocious and I suggest you fix it.

**Comment by pudl** [comment]
CHERIDAN WHY

**Comment by Cheridan** [comment]
mergebeg meme

**Pull request merged**